### ISSUE TITLE
Decide which news item to highlight 

### ISSUE THREAD

**Issue by marinadamato** [DIAGNijmegen/website-content#563]
The main image at the top of the DIAG and CPG websites can either be the latest news or a predefined image. There is no option to select a news item (not the latest) to highlight. We would like to add this feature as we have now started adding the DIAG monthly news to the websites and we don't want this news to be the highlight.

**Comment by marinadamato** [comment]
Added code to:
- Check if the latest news item is a monthly news: if so, the highlight is automatically set to the second-last item.
- Alternatively, there is now the option to specify a specific article to highlight. To do this, update the `pelicanconf.py` file of the website and set `HOME_IMAGE='specific_highlight'` and add a new argument `HIGHLIGHTED_ARTICLE_TITLE` equal to the title of the article.

I updated the [related documentation](https://github.com/DIAGNijmegen/website-content/blob/master/docs/edit-home-page.md).
